community/esetroot/depends

```diff
@@ -0,0 +1,2 @@
+make
```

Comment:
`make` can be removed as it is an implicit dependency. Same as not listing `gcc`, `musl` etc :)